app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/payment_methods/MainPaymentsView.kt

```diff
@@ -0,0 +1,385 @@
+    }
+  }
+}
+
+@Composable
+private fun PaymentMethodView(
+  modifier: Modifier = Modifier,
+  iconUrl: String,
+  label: String,
+  onPaymentMethodClick: () -> Unit,
+) {
+  Row(
+    modifier = modifier
+      .fillMaxWidth()
+      .background(Palette.GreyLight)
+      .padding(16.dp)
+      .clickable(onClick = onPaymentMethodClick),
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    AptoideAsyncImage(
+      modifier = Modifier.size(22.dp),
+      data = iconUrl,
+      contentDescription = null
+    )
+    Text(
+      text = label,
+      maxLines = 2,
+      style = AGTypography.InputsM,
+      color = Palette.Black,
+      overflow = TextOverflow.Ellipsis,
+      modifier = Modifier
+        .padding(horizontal = 16.dp)
+        .weight(1f)
+    )
+    Image(
+      imageVector = getLeftArrow(Palette.Black, Palette.Primary),
+      contentDescription = null,
+      modifier = Modifier
+        .size(24.dp)
+        .rotate(180f)
+    )
+  }
+}
+
+@Composable
+private fun WalletPaymentMethod(
+  onPaymentMethodClick: () -> Unit,
+) {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth()
+      .clickable(onClick = onPaymentMethodClick),
+    horizontalAlignment = Alignment.CenterHorizontally
+  ) {
+    Row(
+      modifier = Modifier
+        .fillMaxWidth()
+        .background(Palette.GreyLight)
+        .padding(16.dp),
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
+      Image(
+        modifier = Modifier.size(22.dp),
+        imageVector = getAppcoinsClearLogo(Palette.Black),
+        contentDescription = null
+      )
+      Column(
+        modifier = Modifier
+          .padding(horizontal = 16.dp)
+          .weight(1f)
+      ) {
+        Text(
+          text = "Pay with the AppCoins Wallet", // TODO hardcoded string
+          maxLines = 1,
+          style = AGTypography.InputsM,
+          color = Palette.Black,
+          overflow = TextOverflow.Ellipsis
+        )
+        Text(
+          text = "It'll be installed automatically and you'll get a bonus in all purchases!", // TODO hardcoded string
+          maxLines = 2,
+          style = AGTypography.Body,
+          overflow = TextOverflow.Ellipsis,
+          color = Palette.Black,
+          modifier = Modifier.padding(end = 10.dp)
+        )
+      }
+      Image(
+        imageVector = getLeftArrow(Palette.Black, Palette.Primary),
+        contentDescription = null,
+        modifier = Modifier
+          .size(24.dp)
+          .rotate(180f)
+      )
+    }
+    Row(
+      modifier = Modifier
+        .padding(horizontal = 16.dp)
+        .fillMaxWidth()
+        .background(Palette.Primary)
+        .padding(vertical = 8.dp),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.Center,
+    ) {
+      Image(
+        imageVector = getWalletGift(Palette.Black),
+        contentDescription = null,
+      )
+      Text(
+        text = "Up to 20% bonus", // TOD hardcoded string
```

Comment:
TOD will not de picked up by the Android Studio.
Put a TODO please 🙂

Reply:
fixed